development/cpp/configuring_an_ide.rst

```diff
@@ -140,6 +92,67 @@ clicking the "Apply Filter" button. A simple click on "Ok" and you're ready to c
 
 .. image:: img/qtcreator-edit-files-dialog.png
 
+Kdevelop
+--------
+
+It is a free, open source IDE for Linux, Solaris, FreeBSD, Mac OS X and other Unix flavors.
```

Comment:
KDevelop also supports Windows, this should be updated:  https://www.kdevelop.org/download

Reply:
Thanks, updated.